### PR TITLE
Pipeline integration

### PR DESCRIPTION
**Pull request by alexmaslenn**
Updated component versions and integrated security for deployment pipeline

### CODE REVIEW

File: openstack/input.yaml [hunk 1]
```diff
@@ -21,8 +21,10 @@ xopera_env:
   XOPERA_VERBOSE_MODE: debug
   XOPERA_DATABASE_IP: xopera-postgres
   XOPERA_DATABASE_NAME: postgres
-  XOPERA_DATABASE_USER: username
-  XOPERA_DATABASE_PASSWORD: password
```
Comment (mihaTrajbaric) [inline openstack/input.yaml]: This should not be removed.

**Review by mihaTrajbaric — CHANGES_REQUESTED** [review]
It looks good, I request a minor change.

However, I had a thought about versions.

I think it is a really good thing to update the components' versions. I think we still have some outdated versions (xopera-rest-api, toscaverifier, workflowverifier,...) and some components are simply set to latest (keycloak, vault, pds,...)

It would be a good thing to control all the versions so we can ensure future deployability of this version of iac-platform-stack

File: openstack/input.yaml (hunk 1 above)
Comment (alexmaslenn) [reply openstack/input.yaml]: Sure thing. IDK how this happened.

**Comment by alexmaslenn** [comment]
@mihaTrajbaric I agree. Lets make a new issue and update all components versions. Some outdated components however were probably not really updated since.

**Comment by mihaTrajbaric** [comment]
> @mihaTrajbaric I agree. Lets make a new issue and update all components versions. Some outdated components however were probably not really updated since.

Ok, I agree

**Review by mihaTrajbaric — APPROVED** [review]
:+1:

**Pull request merged**